src/main/java/io/split/android/client/service/attributes/AttributeClearTask.java

```diff
@@ -0,0 +1,25 @@
+package io.split.android.client.service.attributes;
+
+import androidx.annotation.NonNull;
+
+import io.split.android.client.service.executor.SplitTask;
+import io.split.android.client.service.executor.SplitTaskExecutionInfo;
+import io.split.android.client.service.executor.SplitTaskType;
+import io.split.android.client.storage.attributes.PersistentAttributesStorage;
+
+public class AttributeClearTask implements SplitTask {
```

Comment:
I would be good to indicate in the task name that the task clears from persistent cache...
Perhaps ClearAttributesFromPersistentStorageTask ?